tool/ci.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Created with package:mono_repo v6.5.7
+# Created with package:mono_repo v6.5.1
```

Comment:
I think you needed a `dart pub global activate mono_repo` here to get the latest version locally.